napari/layers/_markers_layer/model.py

```diff
@@ -154,8 +154,12 @@ def symbol(self) -> str:
         return self._symbol
```

Comment:
If you are going forward with enums, this should at least still return a string.
```suggestion
         return str(self._symbol)
```

Reply:
Thanks for the feedback. If I understand your comment, your primary concerns are: (1) you want to keep the ability to set marker symbols with the alternative names via `Markers.symbol` and (2) you don't think using an Enum to hold the deduplicated names is useful.

- Would you prefer that we hold the deduplicated names for the UI as strings in a list or tuple in the _constants.py file? This is a straightforward change, we just have to string match the updated `Markers.symbol` value and the names in the UI dropdown (this is how it used to be).
- We can include a dictionary to map the alias to our deduplicated symbol names in the setter.

Reply:
1. yeah preferably stored as strings in a tuple as a class variable
2. yeah that sounds great